platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/sources/GeoJsonOptions.java

```diff
@@ -109,4 +109,33 @@ public GeoJsonOptions withClusterRadius(int clusterRadius) {
     this.put("clusterRadius", clusterRadius);
     return this;
   }
+
+  /**
+   * An object defining custom properties on the generated clusters if clustering is enabled,
+   * aggregating values from clustered points. Has the form {"property_name": [operator, [map_expression]]} or
+   * {"property_name": [[operator, accumulated, expression], [map_expression]]}
+   *
+   * @param propertyName name of the property
+   * @param operator operator is any expression function that accepts at least 2 operands (e.g. "+" or "max") - it
+   *                 accumulates the property value from clusters/points the cluster contains. It can either be a
+   *                 string with single operator, or be an object array converted from valid expression
+   * @param mapExpr map expression produces the value of a single point, it shall be an object array, which is
+   *                converted from a valid expression
+   * @return the current instance for chaining
+   */
+  @NonNull
+  public GeoJsonOptions withClusterProperty(String propertyName, Object operator, Object mapExpr) {
```

Comment:
could we change the signature of this method to take a strongly typed `Expression` instead of `Object`? In the method body we can call  `Expression#toArray()` without having users of the library needing to call it as shown in the example code in https://github.com/mapbox/mapbox-gl-native/pull/15425/files#diff-f500db5272de0a3199d6f964419ffe60R145

Reply:
For constant values as "+" or "any" the user needs to wrap them with `literal("+")`, this is inline with the rest of the expressions API

Reply:
Got it, I changed them in the new commits.